README.md

```diff
@@ -169,6 +169,141 @@ Controllers from this demo:
   - `Forward Command Controller` ([`ros2_controllers` repository](https://github.com/ros-controls/ros2_controllers)): [doc](https://ros-controls.github.io/control.ros.org/ros2_controllers/forward_command_controller/doc/userdoc.html)
 
 
+### *RRBot* Gazebo Classic simulation
+
+**NOTE**: For this example to work, you have to add the following repository to your workspace:
+
+ - gazebo_ros2_control: `git clone git@github.com:ros-simulation/gazebo_ros2_control.git --branch master`
+
+1. To start *RRBot* gazebo simulation open a terminal, source your ROS2-workspace and execute its launch file with:
+   ```
+   ros2 launch ros2_control_demo_bringup rrbot_gazebo.launch.py
+   ```
+   The launch file loads robot description, starts gazebo, `Joint State Broadcaster` and `Joint Trajectory Controller`.
+
+   If you can see two orange and one black "box" in `Gazebo` everything has started properly.
+
+
+1. Check if the hardware interface loaded properly, by opening another terminal and executing:
+   ```
+   ros2 control list_hardware_interfaces
+   ```
+   You should get:
+   ```
+   command interfaces
+        joint1/position [claimed]
+        joint2/position [claimed]
+   state interfaces
+         joint1/position
+         joint2/position
+
+   ```
+
+1. Check is controllers are running:
+   ```
+   ros2 control list_controllers
+   ```
+   You should get:
+   ```
+   joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+   position_trajectory_controller[joint_trajectory_controller/JointTrajectoryController] active
+   ```
+
+1. If you get output from above you can send commands to *Joint Trajectory Controller*, using script:
```

Comment:
I am not a native speaker but I would reformulate this sentence as:
If you get the output above you can send ...

Reply:
```suggestion
1. If you get the output above you can send commands to *Joint Trajectory Controller*, using script:
```